Documentation/network/bgp-control-plane.rst

```diff
@@ -525,6 +525,16 @@ with a dummy key and value can be used like:
          matchExpressions:
             - {key: somekey, operator: NotIn, values: ['never-used-value']}
 
+There are two special purpose selector fields that match CiliumPodIPPools based on ``name`` and/or
+``namespace`` metadata instead of labels:
+
+=============================== ===================
+Selector                        Field
+------------------------------- -------------------
+io.cilium.podippool.namespace   ``.meta.namespace``
+io.cilium.podippool.name        ``.meta.name``
+=============================== ===================
```

Comment:
Spell checker complains about these - let's quote them to make them literals.

```suggestion
=================================== ===================
Selector                            Field
----------------------------------- -------------------
``io.cilium.podippool.namespace``   ``.meta.namespace``
``io.cilium.podippool.name``        ``.meta.name``
=================================== ===================
```

Reply:
@qmonnet thanks for the review. ^ changes are inconsistent with [existing docs](https://docs.cilium.io/en/latest/network/bgp-control-plane/#service-announcements). Instead of changing the existing docs to resolve the CI spellchecker, I added `podippool` to `Documentation/spelling_wordlist.txt`.

Reply:
I'd rather have the existing docs using inline literals too, it would make more sense than adding them as valid words in my opinion. But OK, works for me.